apps/mail/components/ui/nav-main.tsx

```
@@ -189,6 +190,36 @@ function NavItem(item: NavItemProps & { href: string }) {
+    openFeaturebase();
+  }
+
+  if (item.onClick) {
+    item.onClick(e);
+  }
+};
+
+return (
+  <Collapsible defaultOpen={item.isActive}>
+    <CollapsibleTrigger asChild>
+      <Link {...linkProps} target={item.target} onClick={handleClick}>
```

Biome (1.9.4)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)